src/navigator/Headers.tsx

```diff
@@ -300,6 +295,17 @@ export const tabHeader: NativeStackNavigationOptions = {
   ),
 }
 
+export const headerWithCloseButton: NativeStackNavigationOptions = {
+  ...emptyHeader,
+  headerLeft: () => (
+    // The negative margin is to fix an issue with margin added via the stack navigator
+    // https://github.com/react-navigation/react-navigation/issues/11116
```

Comment:
this issue seems Android specific, but have the same in both iOS and Android?

Reply:
Updated docs in 4bb88204626fa33c4a905591dd76a2dbe79e6229!